lib/house.rb

```diff
@@ -0,0 +1,46 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    @price[1..-1].to_i > 500000 ? true : false
```

Comment:
This is nice, although technically, you don't need the ternary here. 

if you just did:
```
def above_market_average?
     @price[1..-1].to_i > 500000
end
```

it would return the correct boolean all on its own, without you explicitly needing to return true or false